app/static/src/app/components/options/SensitivityOptions.vue

```diff
@@ -124,6 +125,18 @@ export default defineComponent({
             return editSettingsIdx.value === null ? null : allSettings.value[editSettingsIdx.value];
         });
 
+        const paramNamesForSettingsEdit = computed(() => {
+            const allParamNames = store.state.run.parameterValues ? Object.keys(store.state.run.parameterValues) : [];
+            if (props.multiSensitivity) {
+                // return all params which are either unused or are the param for the settings to edit
+                const editIdx = editSettingsIdx.value;
+                const editParam = editIdx !== null && (allSettings.value[editIdx]?.parameterToVary || null);
+                return allParamNames.filter((p) => paramsWithoutSettings.value.includes(p)
+                    || (!addingParamSettings.value && editParam !== null && p === editParam));
```

Comment:
I found that line a little dense to understand. I think if you just want to include this extra bit, it is cleaner to just include it somewhere instead of that filter condition! Diff in this suggestion is being a bit weird for me here btw
```suggestion
                const editIdx = editSettingsIdx.value;
                const editParam = editIdx !== null && (allSettings.value[editIdx]?.parameterToVary || null);
                const paramNamesWithoutSettings = allParamNames.filter((p) => paramsWithoutSettings.value.includes(p));
                if (!addingParamSettings.value && editParam !== null) {
                    // include the param we are currently editting
                    return [editParam, ...paramNamesWithoutSettings];
                }
                return paramNamesWithoutSettings;
```

Reply:
I think the reason I did it this way was that I didn't want to disrupt the order of the parameters in the drop down - I'd like to keep the current param in its usual place rather than sticking it in the beginning, so letting the filter handle that seemed like a good way to retain order without making any assumptions about what that order might be. I should still be able to break up that dense line a bit though.